pt-BR/mail/chrome/messenger/aboutSupportMail.dtd

```diff
@@ -1,26 +1,26 @@
-<!ENTITY aboutSupport.appBasicsAppBuildID "Application Build ID">
+<!ENTITY aboutSupport.appBasicsAppBuildID "ID da versão da aplicação">
 
-<!ENTITY aboutSupport.accountsTitle "Mail and News Accounts">
-<!ENTITY aboutSupport.showPrivateData.mainText "Include account names">
-<!ENTITY aboutSupport.showPrivateData.explanationText "(possibly identifying information)">
+<!ENTITY aboutSupport.accountsTitle "E-mail e novas contas">
+<!ENTITY aboutSupport.showPrivateData.mainText "Incluir nomes das contas">
+<!ENTITY aboutSupport.showPrivateData.explanationText "(provavelmente identificando informações)">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY aboutSupport.showPrivateData.explanationText "(informações que podem revelar identidade)">